app.py

```diff
@@ -1,10 +1,11 @@
 from bottle import request, route, run, template, post, TEMPLATE_PATH, abort, redirect
 from marshmallow import Schema, fields
-import dbfunctions
+from dbfunctions import Wikidb
 
 TEMPLATE_PATH.append('templates')
 DB_PATH = './wiki.db'
 
+db = Wikidb(db_path=DB_PATH)
```

Comment:
What's the reason for moving this here (used to be on old line 65 as `dbfunctions.init_db(DB_PATH)`)?

I still feel pretty strongly about not having the DB created on import _without a good reason_.  Personally, I don't like seeing **any** code executed on import that doesn't really need to be there.  I've been bitten by this quite a bit in the past, so I'm a bit sensitive :).